test/com/google/enterprise/adaptor/HeartbeatHandlerTest.java

```diff
@@ -272,28 +270,32 @@ public void getDocContent(Request request, Response response)
 
   @Test
   public void testWatchdogNoInterruption() throws Exception {
-    ScheduledExecutorService executor
-        = Executors.newSingleThreadScheduledExecutor();
+    ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
     Watchdog watchdog = new Watchdog(executor);
     Watchdog watchdog2 = new Watchdog(executor);
-    mockAdaptor = new MockAdaptor() {
-      @Override
-      public void getDocContent(Request request, Response response)
-          throws IOException, InterruptedException {
-        Thread.sleep(100);
-        super.getDocContent(request, response);
-      }
-    };
-    DocumentHandler docHandler = createDocHandlerBuilder()
-        .setAdaptor(mockAdaptor)
-        .setAuthzAuthority(mockAdaptor)
-        .setWatchdog(watchdog)
-        .setHeaderTimeoutMillis(100)
-        .setContentTimeoutMillis(100)
-        .build();
-    HeartbeatHandler handler = createHeartbeatHandlerBuilder()
-        .setWatchdog(watchdog2).setDocHandler(docHandler)
-        .build();
+    mockAdaptor =
```

Comment:
This is why John is a stickler for arbitrary whitespace changes.  It took me far too long to detect that only the timeouts have changed.